.github/workflows/book-pages.yml

```
@@ -1,34 +1,34 @@
-    uses: ryancinsight/atlas/.github/workflows/book-pages.yml@1fcd17c6f7923cb1734756c15e0a5a39e333ee32
-    permissions:
-      contents: read
-      pages: write
-      id-token: write
-    with:
-      output-path: target/book/moirai
-      mdbook-test: true
-      rust-toolchain: "1.97.0"
-      cargo-package: moirai-runtime
-      cargo-crate: moirai
+name: Deploy mdBook
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)